drivers/ukbus/pci/pci_ecam.c

```diff
@@ -234,7 +234,7 @@ int gen_pci_irq_parse(const fdt32_t *addr, struct fdt_phandle_args *out_irq)
 	const fdt32_t *prop;
 
 	ipar = gen_pci_fdt;
-	memset((void *)dummy_imask, cpu_to_fdt32(~0), ARRAY_SIZE(dummy_imask));
+	memset((void *)dummy_imask, cpu_to_fdt32(~0), sizeof(dummy_imask));
```

Comment:
commit message headline too long, it appears as
```
drivers/ukbus/pci: Fix initialization of dummy_imask in gen_pci_irq_p…
…arse
```

I see that you added
```
Checkpatch-Ignore: COMMIT_LOG_LONG_LINE
```
Maybe it would be worth re-wording it though as this is too generic and not specific enough. How about:
```
drivers/ukbus/pci: Memset `dummy_imask` by size not unit count
```
I've also been met with similar situations quite frequently unfortunately :disappointed:.

Reply:
Actually the headline is 73 chars, while the checkpatch limit is 75. The problem is the error message of GCC. I will wrap the long line (~120 chars) so that the max line is 80 chars.

Reply:
> Actually the headline is 73 chars, while the checkpatch limit is 75.

Which is still above 70, the maximum recommended for headlines AFAIK.

> The problem is the error message of GCC. I will wrap the long line (~120 chars) so that the max line is 80 chars.

I would actually leave that as is. From what I have seen on mailing lists, usually errors/warnings/crashdumps/etc are kept without wrapping

Reply:
This is the relevant snippet from checkpatch:
```
# Check for line lengths > 75 in commit log, warn once
                if ($in_commit_log && !$commit_log_long_line &&
                    length($line) > 75 &&
                    !($line =~ /^\s*[a-zA-Z0-9_\/\.]+\s+\|\s+\d+/ ||
                                        # file delta changes
                      $line =~ /^\s*(?:[\w\.\-]+\/)++[\w\.\-]+:/ ||
                                        # filename then :
                      $line =~ /^\s*(?:Fixes:|Link:)/i ||
                                        # A Fixes: or Link: line
                      $commit_log_possible_stack_dump)) {
                        WARN("COMMIT_LOG_LONG_LINE",
                             "Possible unwrapped commit description (prefer a maximum 75 chars per line)\n" . $herecurr);
                        $commit_log_long_line = 1;
                }
```

>  I would actually leave that as is. From what I have seen on mailing lists, usually errors/warnings/crashdumps/etc are kept without wrapping

Yes, that was my initial rationale. Ok then I'll leave it and we can blame GCC for spewing such long messages.